src/Serializer/AbstractItemNormalizer.php

```diff
@@ -186,10 +186,19 @@ public function denormalize($data, $class, $format = null, array $context = [])
             if (!$this->serializer instanceof DenormalizerInterface) {
                 throw new LogicException('Cannot denormalize the input because the injected serializer is not a denormalizer');
             }
+            if ($dataTransformer instanceof PreHydrateInputInterface) {
+                $context[static::OBJECT_TO_POPULATE] = $dataTransformer->createInput($inputClass, $context);
```

Comment:
I'm not sure to understand what you're trying to do here, this doesn't look to go the way we meant data transformers to work, could you explain your use case?

Reply:
I set this key value by the object we want to populate by the symfony serializer (https://symfony.com/doc/current/components/serializer.html#deserializing-in-an-existing-object)